### PR TITLE
#65: Replace env variables by properties. Support gradle docker plugi…

### PR DESCRIPTION
**Pull request by dartartem**
…n. Removed redundant env files, fixed eventuate cdc service name in publishing script.

### CODE REVIEW

File: scripts/deploy-artifacts.sh [hunk 1]
```diff
@@ -58,4 +58,4 @@ $PREFIX docker login -u ${DOCKER_USER_ID?} -p ${DOCKER_PASSWORD?}
 
 docker images
 
-tagAndPush "cdcservice" "eventuate-cdc-service"
+tagAndPush "eventuate-cdc-service" "eventuate-cdc-service"
```
Comment (cer) [inline scripts/deploy-artifacts.sh:61]: This should be done by gradlew composePush - not by docker push

The docker-compose file should define the images to push.

**Review by cer — CHANGES_REQUESTED** [review]

File: scripts/deploy-artifacts.sh (hunk 1 above)
Comment (dartartem) [reply scripts/deploy-artifacts.sh:61]: ok, I will fix

**Comment by cer** [comment]
Is it ready to merge?

**Comment by dartartem** [comment]
@cer it is ready to merge

**Pull request merged**